<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AGMEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Grammar View</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QWidget" name="lhsParentWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QWidget" name="rhsParentWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Left Hand Side</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Right Hand Side</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Behavior view</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionGraphmar"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionChangeAppearance"/>
    <addaction name="actionChangeFont"/>
   </widget>
   <widget class="QMenu" name="menuRules">
    <property name="title">
     <string>Rules</string>
    </property>
    <addaction name="actionAddRule"/>
    <addaction name="actionRemoveCurrentRule"/>
    <addaction name="actionRenameCurrentRule"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionExportAllRules"/>
    <addaction name="actionExportAllRulesPNG"/>
    <addaction name="actionGenerateCode"/>
   </widget>
   <widget class="QMenu" name="menuBehaviors">
    <property name="title">
     <string>Behaviors</string>
    </property>
    <addaction name="actionNew_configuration"/>
    <addaction name="actionRemove_current_config"/>
    <addaction name="actionNew_agent"/>
    <addaction name="actionRemove_current_agent"/>
    <addaction name="actionNew_agent_state"/>
    <addaction name="actionRemove_current_agent_state"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRules"/>
   <addaction name="menuBehaviors"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="productionsWidget">
   <property name="windowTitle">
    <string>      Rules</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="rulesList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="toolsWidget">
   <property name="windowTitle">
    <string>      Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="toolsList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="behaviorsWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>      Configuration name</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QListWidget" name="configurationListWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="agentsDock">
   <property name="windowTitle">
    <string>      Agents</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QListWidget" name="agentListWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="agentStatesDock">
   <property name="windowTitle">
    <string>      Agent states</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QListWidget" name="agentStatesListWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionChangeAppearance">
   <property name="text">
    <string>Change node appearance</string>
   </property>
  </action>
  <action name="actionChangeWidth">
   <property name="text">
    <string>Change width</string>
   </property>
  </action>
  <action name="actionChangeFont">
   <property name="text">
    <string>Change font</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionGraphmar">
   <property name="text">
    <string>AGMEditor</string>
   </property>
  </action>
  <action name="actionAddRule">
   <property name="text">
    <string>Add new rule</string>
   </property>
  </action>
  <action name="actionRemoveCurrentRule">
   <property name="text">
    <string>Remove current rule</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export to SVG current rule</string>
   </property>
  </action>
  <action name="actionExportAllRules">
   <property name="text">
    <string>Export to SVG all rules</string>
   </property>
  </action>
  <action name="actionGenerateCode">
   <property name="text">
    <string>Generate C++ code</string>
   </property>
  </action>
  <action name="actionRenameCurrentRule">
   <property name="text">
    <string>Rename current rule</string>
   </property>
  </action>
  <action name="actionNew_agent">
   <property name="text">
    <string>New agent</string>
   </property>
  </action>
  <action name="actionRemove_current_agent">
   <property name="text">
    <string>Remove current agent</string>
   </property>
  </action>
  <action name="actionNew_configuration">
   <property name="text">
    <string>New configuration</string>
   </property>
  </action>
  <action name="actionRemove_current_config">
   <property name="text">
    <string>Remove current configuration</string>
   </property>
  </action>
  <action name="actionNew_agent_state">
   <property name="text">
    <string>New agent state</string>
   </property>
  </action>
  <action name="actionRemove_current_agent_state">
   <property name="text">
    <string>Remove current agent state</string>
   </property>
  </action>
  <action name="actionExportAllRulesPNG">
   <property name="text">
    <string>Export to PNG all rules</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
